source/tutorials/read-joining.rst

```diff
@@ -0,0 +1,223 @@
+microbiome tutorial <atacama-soils>`.
+
+.. download::
+   :url: https://data.qiime2.org/2018.2/tutorials/read-joining/atacama-seqs.qza
+   :saveas: demux.qza
+
+Joining reads
+~~~~~~~~~~~~~
+
+Next, use the ``join-pairs`` method in the ``q2-vsearch`` plugin to join the
+reads:
+
+.. command-block::
+
+   qiime vsearch join-pairs \
+     --i-demultiplexed-seqs demux.qza \
+     --o-joined-sequences demux-joined.qza
+
+Viewing a summary of joined data with read quality
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can next generate a summary of the ``demux-joined.qza`` artifact.
+
+.. command-block::
+
+   qiime demux summarize \
+     --i-data demux-joined.qza \
+     --o-visualization demux-joined.qzv
+
+This summary is particularly useful for determining approximately how long your
+joined reads are (we’ll come back to this when we denoise with Deblur). When
+looking at the quality plots in this visualization, if you hover over a
+specific position you’ll see how many reads are at least that long (of the
+reads that were sampled for computing sequence quality). Make note of the
+highest sequence position where most (say, > 99%) of your reads are at least
+that long.
+
+For example, when hovering over a black box in this visualization (which is
+generated from a larger data set than the one used in this tutorial), I see
+that 10000 out of the 40126 sequences were used to estimate the quality score
+distribution at this position.
+
+When I hover over position 250, which is illustrated with a red box, I see that
+some sequences are not this long because only 9994 sequences were used for
+estimating the quality score distribution at this position. The red box and the
+red text below tell me that some sequences were not at least this long.
+
+When I hover over position 254, which is also illustrated with a red box, I see
+that many sequences are not this long because only 845 sequences were used for
+estimating the quality score distribution at this position.
+
+**Based on a comparison of these plots, I will note that most of my sequences
+are at least 250 bases long.** We plan to simplify this process in `the near
+future`_.
+
+Sequence quality control
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next we’ll apply quality control to our sequences using ``quality-filter
+q-score-joined``. This method is identical to ``quality-filter q-score``,
+except that it operated on joined reads. The parameters to this method have not
+been extensively benchmarked on joined read data, so we recommend experimenting
+with different parameter settings.
+
+.. command-block::
+
+   qiime quality-filter q-score-joined \
+     --i-demux demux-joined.qza \
+     --o-filtered-sequences demux-joined-filtered.qza \
+     --o-filter-stats demux-joined-filter-stats.qza
+
+At this stage you can choose to proceed using `Deblur`_ for additional quality
+control, or you can dereplicate sequences and optionally cluster them into OTUs
```

Comment:
provide a link? or perhaps you are subtly discouraging irresponsible use of vsearch... friends don't let friends pick otus? 🍻 🚗 💥